src/sentry/workflow_engine/migration_helpers/rule.py

```diff
@@ -110,3 +115,110 @@ def create_if_dcg(
 def create_workflow_actions(if_dcg: DataConditionGroup, actions: list[dict[str, Any]]):
     # TODO: create actions, need registry
     pass
+
+
+def update_migrated_issue_alert(rule: Rule, data: dict[str, Any], user: RpcUser | None = None):
+    try:
+        alert_rule_workflow = AlertRuleWorkflow.objects.get(rule=rule)
+    except AlertRuleWorkflow.DoesNotExist:
+        logger.exception("AlertRuleWorkflow does not exist", extra={"rule_id": rule.id})
+        return
+
+    workflow: Workflow = alert_rule_workflow.workflow
+    if not workflow.when_condition_group:
+        # won't let me throw a logger exception here
+        return
+
+    conditions, filters = split_conditions_and_filters(data["conditions"])
+
+    update_when_dcg(
+        when_dcg=workflow.when_condition_group,
+        conditions=conditions,
+        action_match=data["action_match"],
+        user=user,
+    )
+
+    try:
+        if_dcg = WorkflowDataConditionGroup.objects.get(workflow=workflow).condition_group
+    except WorkflowDataConditionGroup.DoesNotExist:
+        logger.exception(
+            "WorkflowDataConditionGroup does not exist", extra={"workflow_id": workflow.id}
+        )
+        # if no IF DCG exists, create one
+        if_dcg = create_if_dcg(
+            workflow=workflow, filters=filters, filter_match=data.get("filter_match")
+        )
+        WorkflowDataConditionGroup.objects.create(workflow=workflow, condition_group=if_dcg)
+
+    update_if_dcg(if_dcg=if_dcg, filters=filters, filter_match=data.get("filter_match"), user=user)
+
+    delete_workflow_actions(if_dcg=if_dcg, actions=data["actions"], user=user)
+    create_workflow_actions(if_dcg=if_dcg, actions=data["actions"])  # action(s) must exist
+
+    updated_data = {
+        "name": data["name"],
+        "environment_id": data["environment"],
+        "enabled": True,
+        "owner_team_id": None,
+        "owner_user_id": None,
+        "config": {"frequency": data.get("frequency", 30)},
```

Comment:
can we use a constant for `30`? this is also used in the create helper, maybe we can attach it to `Workflow` as `Workflow.DEFAULT_FREQUENCY`